library/core/src/cmp.rs

```diff
@@ -1391,17 +1391,17 @@ mod impls {
             #[stable(feature = "rust1", since = "1.0.0")]
             #[rustc_const_unstable(feature = "const_cmp", issue = "92391")]
             impl const PartialOrd for $t {
-                #[inline]
+                #[inline(always)]
                 fn partial_cmp(&self, other: &$t) -> Option<Ordering> {
                     Some(self.cmp(other))
```

Comment:
This is dozens of lines of MIR, due to being a bunch of primitive operations and switches (<https://rust.godbolt.org/z/jaazhve7E>), so it's not clear to me that it should be `always`'d.

It probably wants #105840 or a spaceship intrinsic to cut down the MIR size before doing it *always*.